sklearn/linear_model/_logistic.py

```diff
@@ -1938,6 +1938,10 @@ class LogisticRegressionCV(LogisticRegression, LinearClassifierMixin, BaseEstima
 
         .. versionadded:: 0.24
 
+    See Also
+    --------
+    LogisticRegression : Logistic regression.
```

Comment:
```suggestion
    LogisticRegression : Logistic regression without tuning the hyperparameter `C`.
```

Maybe this line is too long. You might need to rewrap the line.

Reply:
Hi @glemaitre , thank you for the feedback!
How do I know a line is too long? Is there a convention, besides running flake8 and black?

Thank you!